recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "0.1.1" %}
+{% set version = "0.1.2a2" %}
```

Comment:
@CJ-Wright and @justcalamari we need to prevent the bot from adding pre-releases or we'll be swamped with bleeding edge packages!

Reply:
@justcalamari could you please add 'a' and 'b' to the version backlist? This most likely needs more discussion (which have me hesitation before) but we most likely should default to the safer course when in doubt. There's nothing stopping us from reversing this policy down the line if that is desirable.

Reply:
IMO we are working around bad practices here. Since PyPI allowed for pre-releases devs are using that instead of releasing early/releasing often, and users are now relying on pre-releases for production work.

Besides that cultural change we still have the fact that conda does not behave like pip! pre-releases on the main channel will clobber stable releases when installing/updating a package. And putting them in dev label is not ideal either :-/

TL;DR until conda behaves more like pip we need to block these on the main channel.

Reply:
I don't know how much we want to behave like pip :smiling_imp:

Reply:
When it comes to dealing with pre-releases I say we have too. We cannot change the culture of publishing them, so we need to deal with it by not installing them by default, like pip :-(

Reply:
Hmm ok